packages/server/src/api/routes/view.ts

```diff
@@ -3,23 +3,13 @@ import * as viewController from "../controllers/view"
 import * as rowController from "../controllers/row"
 import authorized from "../../middleware/authorized"
 import { paramResource } from "../../middleware/resourceId"
-import { DocumentType, SEPARATOR, permissions } from "@budibase/backend-core"
+import { permissions } from "@budibase/backend-core"
 
 const router: Router = new Router()
 
 router
-  .get(
-    "/api/v2/views",
-    authorized(permissions.BUILDER),
-    viewController.v2.fetch
-  )
-  .get(
-    `/api/v2/views/:viewId`,
-    authorized(permissions.BUILDER),
-    viewController.v2.find
-  )
   .post(
-    "/api/v2/views",
+    "/api/v2/views/:tableId",
```

Comment:
should this be `:viewId`

Reply:
This was meant to be table, as it's creating the view on this table. But it could be inferred from the body